### DISCUSSION TITLE
摩登犀牛 模板问题反馈评论区

### DISCUSSION THREAD

**Discussion by wjf0214** [wjf0214/qd-templates#88] [模板讨论]
摩登犀牛 模板问题反馈评论区
有问题可以下面反馈。

**Comment by miaoxiaozhao** [comment]
Failed at 3/8 request,
Fail assert: {"re": "错误", "from": "content"} from failed_asserts,
Request URL: http://bbs.rhino3d.us/member.php?mod=logging&action=login&loginsubmit=yes&handlekey=login&loginhash={{loginhash|urlencode}}&inajax=1
